Integrations/MimecastV2/MimecastV2.py

```diff
@@ -443,43 +448,169 @@ def create_policy():
         'toValue': to_value
     }
 
+    return policy_obj, option
+
+
+def create_policy(args):
+    headers = ['Policy ID', 'Description', 'Sender', 'Reciever', 'Bidirectional', 'Start', 'End']
+    contents = {}  # type: Dict[Any, Any]
+    context = {}
+    policies_context = {}  # type: Dict[Any, Any]
+    policy_obj, option = get_arguments_for_policy_command(args)
     policy_list = create_policy_request(policy_obj, option)
     policy = policy_list.get('policy')
     policy_id = policy_list.get('id')
-    title = 'Mimecast Create Policy: \n Policy {} Was Created Successfully!'.format(policy_id)
+    title = 'Mimecast Create Policy: \n Policy Was Created Successfully!'
     sender = policy.get('from')
-    reciever = policy.get('to')
+    receiver = policy.get('to')
+    description = policy.get('description')
+    raw_response = {
+        'Policy ID': policy_id,
+        'Description': description,
+        'Sender': {
+            'Group': sender.get('groupId'),
+            'Email Address': sender.get('emailAddress'),
+            'Domain': sender.get('emailDomain'),
+            'Type': sender.get('type')
+        },
+        'Reciever': {
+            'Group': receiver.get('groupId'),
+            'Email Address': receiver.get('emailAddress'),
+            'Domain': receiver.get('emailDomain'),
+            'Type': receiver.get('type')
+        },
+        'Bidirectional': policy.get('bidirectional'),
+        'Start': policy.get('fromDate'),
+        'End': policy.get('toDate')
```

Comment:
why `start` and `date` instead of FromDate and ToDate in the context?
And if this change isn't needed - this is a duplication of the policies context